src/Admin/AbstractAdmin.php

```diff
@@ -2712,6 +2712,11 @@ public function hasAccess($action, $object = null)
      */
     public function configureActionButtons($action, $object = null)
     {
+        // nothing to do for non-internal actions
+        if (!\in_array($action, ['tree', 'show', 'edit', 'delete', 'acl', 'history', 'list', 'batch'], true)) {
+            return [];
+        }
+
```

Comment:
All the following code for this method does not make sense for custom actions. Perhaps it would be better to immediately identify these custom actions?